Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM centos:7.4.1708
 
 # ARG can be overwritten on build time using "docker build --build-arg name=value"
-ARG CMK_VERSION_ARG="1.4.0p33"
+ARG CMK_VERSION_ARG="1.5.0p1"
 ARG CMK_DOWNLOADNR_ARG="80"
```

Comment:
This variable must be set to the number found in the link for the CentOS 7 download (here: 38)